cfme/tests/automate/custom_button/test_generic_objects.py

```diff
@@ -345,4 +361,39 @@ def test_custom_button_open_url_evm_obj(setup_obj, button_group):
     Bugzilla:
         1550002
     """
-    pass
+    group, obj_type = button_group
+
+    button = group.buttons.create(
```

Comment:
Q: thoughts on moving this into a fixture?

Reply:
@john-dupuy our test is related to `open_url`. we are enabling `open url` while creating button [L369].
So as per automation practice; whatever, we want to test better to make part of test `steps`.